examples/diet.jl

```diff
@@ -95,7 +95,7 @@ function example_diet(; verbose = true)
     @constraint(model, buy["milk"] + buy["ice cream"] <= 6)
     verbose && println("Solving dairy-limited problem...")
     JuMP.optimize!(model)
-    @test JuMP.termination_status(model) == MOI.INFEASIBLE_OR_UNBOUNDED
+    @test JuMP.termination_status(model) == MOI.INFEASIBLE
     @test JuMP.primal_status(model) == MOI.NO_SOLUTION
```

Comment:
@odow It's correct that GLPK now returns a more specialized status code, isn't it?

Reply:
Yes. I clarified a few of the statuses.